### PR TITLE
Added Gerson Esquivel

### PR DESCRIPTION
**Pull request by GersonEsquivel**

### CODE REVIEW

File: _data/people.yml [hunk 1]
```diff
@@ -47,6 +47,9 @@
 - group: 'Undergraduate Students'
   people:
   - name: "Bowen Quan"
+  - name: "Gerson A. Esquivel Garcia"
+    url: "gaeg"
+    image: Gerson-3 copy.jpg
```
Comment (gonuke) [inline _data/people.yml:52]: These lines should be a new entry, ideally after Damian's below

File: community/people/gaeg/index.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+layout: person
+title: Gerson Adimir Esquivel Garcia
+role: Undergraduate Student
+image: Gerson-3 copy.jpg
+email: esquivelgarc@wisc.edu
+phone: "608/4194316"
+city_state_zip: "Madison, WI 53706"
+services:
+  github: GersonEsquivel
+  facebook: Gerson Esquivel Garcia
```
Comment (gonuke) [inline community/people/gaeg/index.md:11]: I think this should be the value for direct access to your page via a facebook url:
```suggestion
  facebook: gerson.esquivel.garcia
```

**Review by gonuke — CHANGES_REQUESTED** [review]
Thanks for adding yourself to these pages @GersonEsquivel !!!

I have a couple of comments to make sure everything works properly.

File: _data/people.yml (hunk 1 above)
Comment (GersonEsquivel) [reply _data/people.yml]: Fixed, I did not see the file was had text beyond what the screen showed.

File: community/people/gaeg/index.md (hunk 2 above)
Comment (GersonEsquivel) [reply community/people/gaeg/index.md]: Fixed

**Comment by gonuke** [comment]
Thanks @GersonEsquivel - This looks better. I'll run a quick test later and confirm before merging.

File: community/people/gaeg/index.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+---
+layout: person
+title: Gerson Adimir Esquivel Garcia
+role: Undergraduate Student
+image: Gerson-3 copy.jpg
```
Comment (gonuke) [inline community/people/gaeg/index.md:5]: Since you changed the name of the file, you need to update it here too

**Review by gonuke — CHANGES_REQUESTED** [review]
Looks like you need to update the name of the file for your photo in one more place.

File: community/people/gaeg/index.md (hunk 3 above)
Comment (GersonEsquivel) [reply community/people/gaeg/index.md]: Right. Sorry about that. Fixed

**Review by gonuke — APPROVED** [review]

**Comment by gonuke** [comment]
Looks great - merging now - thanks for the update @GersonEsquivel

**Pull request merged**